dotenv

```diff
@@ -1 +1,3 @@
-DATABASE_URL=DATABASE_URL
\ No newline at end of file
+DATABASE_URL=DATABASE_URL
+FIREBASE_SERVICE_ACCOUNT_CRED=FIREBASE_SERVICE_ACCOUNT_CRED 
```

Comment:
Is there a purpose for this variable`FIREBASE_SERVICE_ACCOUNT_CRED`?

Reply:
O ya this was from the middleware stuff I removed will update ty